frontend/public/texts/getHubTexts.js

```diff
@@ -132,6 +132,22 @@ export default function getHubTexts({ hubName, hubAmbassador }) {
       en: "All Locations",
       de: "Alle Orte",
     },
+    sign_up_today_and_help_make_climate_neutral: {
+      en: `Sign up today and help to make ${hubName} climate neutral.`,
```

Comment:
Honest question we might want to codify / standardize: do we always include punctuation in the translations? E.g., this `.`. I think it'd be useful to pick one way or the other: always include, or never include. WDYT?